draft-ietf-ice-pac.xml

```diff
@@ -128,7 +129,7 @@
         </t>
         <section title="No Candidates From Peer">
           <t>
-            It is entirely legal for an ICE agent to provide zero candidates of
+            Per RFC XXXX, an ICE agent could provide zero candidates of
```

Comment:
could -> may

Reply:
Is "might" ok? Some people want to avoid 2119 terminology when not used upper-case.